### PR TITLE
Prep for v1.7.0

### PR DESCRIPTION
**Pull request by odow**
Lots of new stuff for ComplexOptInterface that would be good to get out before JuMP-dev.

Closes https://github.com/jump-dev/MathOptInterface.jl/pull/1963

### CODE REVIEW

**Comment by blegat** [comment]
I'd like to squeeze in hermitian PSD as well, expect PRs today

**Review by blegat — APPROVED** [review]
Actually let's do it because I need https://github.com/jump-dev/MathOptInterface.jl/pull/1941

File: docs/src/release_notes.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Release notes
 
+## v1.7.0 (July 25, 2022)
```
Comment (blegat) [inline docs/src/release_notes.md:3]:
```suggestion
## v1.7.0 (July 23, 2022)
```

**Comment by odow** [comment]
I might be going out today, so feel free to merge and tag when this passes.

**Comment by blegat** [comment]
Actually, let's do https://github.com/jump-dev/MathOptInterface.jl/pull/1963 instead

**Comment by blegat** [comment]
Let's have a tests with SolverTests: https://github.com/jump-dev/SolverTests/actions/runs/2725207871

**Pull request merged**